src/xcode/ENA/ENAUITests/AntigenTestProfile/ENAUITests_12_AntigenTestProfile.swift

```diff
@@ -0,0 +1,142 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+
+class ENAUITests_12_AntigenTestProfile: XCTestCase {
+	
+	// MARK: - Overrides
+	
+	override func setUp() {
+		super.setUp()
+		continueAfterFailure = false
+		app = XCUIApplication()
+		app.setDefaults()
+		app.launchArguments.append(contentsOf: ["-isOnboarded", "YES"])
+		app.launchArguments.append(contentsOf: ["-setCurrentOnboardingVersion", "YES"])
+		app.launchArguments.append(contentsOf: ["-antigenTestProfileInfoScreenShown", "NO"])
+		app.launchArguments.append(contentsOf: ["-removeAntigenTestProfile", "YES"])
+	}
+	
+	// MARK: - Internal
+
+	var app: XCUIApplication!
+	
+	// MARK: - Tests
+	
+	func test_FIRST_CreateAntigenTestProfile_THEN_DeleteProfile() throws {
+		
```

Comment:
Do you think it make sense to capture two/three snapshots during testing?

Reply:
I have no idea. Nothing was specified in the story.